eslint-plugin/src/index.js

```diff
@@ -0,0 +1,44 @@
+
+var TSDocParser = require("@microsoft/tsdoc/lib/index").TSDocParser;
+var allTsdocMessageIds = require("@microsoft/tsdoc/lib/parser/TSDocMessageId").allTsdocMessageIds;
+
+var messageIds = {};
+
+allTsdocMessageIds.forEach(function (messageId) {
+    messageIds[messageId] = messageId;
+});
+
+module.exports = {
+    rules: {
+        "tsdoc-comments": {
+            meta: {
```

Comment:
We should include at least the `type` and `docs` properties here (https://github.com/eslint/eslint/blob/master/docs/developer-guide/working-with-rules.md#rule-basics).